src/etc/inc/filter.inc

```diff
@@ -2313,7 +2313,9 @@ function filter_generate_user_rule_arr($rule) {
 	$line = filter_generate_user_rule($rule);
 	$ret['rule'] = $line;
 	$ret['interface'] = $rule['interface'];
-	if ($rule['descr'] != "" and $line != "") {
+	if ($rule['tracker'] != "" and $line != "") {
+		$ret['descr'] = "label \"" . fix_rule_label("USER_RULE: {$rule['tracker']}") . "\"";
```

Comment:
@loos-br is checking the possibility of implementing it in C, as part of pfSense PHP module. This way we would kill that code you added that calls exec(pfctl -vvsr) and will end up calling pfSense_get_stats($rule_tracker) or something like this.